[2024-07-15T23:59:22.400+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-07-15T23:59:22.408+0000] {__init__.py:51} DEBUG - Loading core task runner: StandardTaskRunner
[2024-07-15T23:59:22.414+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-07-15T23:59:22.415+0000] {base_task_runner.py:68} DEBUG - Planning to run as the  user
[2024-07-15T23:59:22.416+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_stg_accounts manual__2024-07-15T23:54:38.113182+00:00 [queued]> from DB
[2024-07-15T23:59:22.425+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_stg_accounts manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-07-15T23:59:22.431+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_stg_accounts manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2024-07-15T23:59:22.433+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_stg_accounts manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-07-15T23:59:22.433+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_stg_accounts manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2024-07-15T23:59:22.433+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_stg_accounts manual__2024-07-15T23:54:38.113182+00:00 [queued]>
[2024-07-15T23:59:22.433+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_stg_accounts manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-07-15T23:59:22.437+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_stg_accounts manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2024-07-15T23:59:22.438+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_stg_accounts manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-07-15T23:59:22.438+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_stg_accounts manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2024-07-15T23:59:22.438+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_stg_accounts manual__2024-07-15T23:54:38.113182+00:00 [queued]>
[2024-07-15T23:59:22.438+0000] {taskinstance.py:1308} INFO - Starting attempt 3 of 4
[2024-07-15T23:59:22.443+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): test.matt_demo.crm_stg_accounts> on 2024-07-15 23:54:38.113182+00:00
[2024-07-15T23:59:22.445+0000] {standard_task_runner.py:57} INFO - Started process 1567 to run task
[2024-07-15T23:59:22.446+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'run_dbt_model_crm_sales', 'test.matt_demo.crm_stg_accounts', 'manual__2024-07-15T23:54:38.113182+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt_models_crm_sales.py', '--cfg-path', '/tmp/tmpjf4hbfcn']
[2024-07-15T23:59:22.447+0000] {standard_task_runner.py:85} INFO - Job 128: Subtask test.matt_demo.crm_stg_accounts
[2024-07-15T23:59:22.447+0000] {cli_action_loggers.py:65} DEBUG - Calling callbacks: [<function default_action_log at 0xffff7ee0a550>]
[2024-07-15T23:59:22.468+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-07-15T23:59:22.470+0000] {task_command.py:410} INFO - Running <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_stg_accounts manual__2024-07-15T23:54:38.113182+00:00 [running]> on host f934d9af8cb8
[2024-07-15T23:59:22.470+0000] {settings.py:353} DEBUG - Disposing DB connection pool (PID 1567)
[2024-07-15T23:59:22.470+0000] {settings.py:212} DEBUG - Setting up DB connection pool (PID 1567)
[2024-07-15T23:59:22.470+0000] {settings.py:285} DEBUG - settings.prepare_engine_args(): Using NullPool
[2024-07-15T23:59:22.471+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_stg_accounts manual__2024-07-15T23:54:38.113182+00:00 [running]> from DB
[2024-07-15T23:59:22.486+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-07-15T23:59:22.487+0000] {taskinstance.py:868} DEBUG - Clearing XCom data
[2024-07-15T23:59:22.498+0000] {retries.py:80} DEBUG - Running RenderedTaskInstanceFields._do_delete_old_records with retries. Try 1 of 3
[2024-07-15T23:59:22.503+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='run_dbt_model_crm_sales' AIRFLOW_CTX_TASK_ID='test.matt_demo.crm_stg_accounts' AIRFLOW_CTX_EXECUTION_DATE='2024-07-15T23:54:38.113182+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-15T23:54:38.113182+00:00'
[2024-07-15T23:59:22.504+0000] {__init__.py:117} DEBUG - Preparing lineage inlets and outlets
[2024-07-15T23:59:22.504+0000] {__init__.py:155} DEBUG - inlets: [], outlets: []
[2024-07-15T23:59:22.504+0000] {bash.py:186} DEBUG - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='run_dbt_model_crm_sales' AIRFLOW_CTX_TASK_ID='test.matt_demo.crm_stg_accounts' AIRFLOW_CTX_EXECUTION_DATE='2024-07-15T23:54:38.113182+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-15T23:54:38.113182+00:00'
[2024-07-15T23:59:22.504+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-07-15T23:59:22.505+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n            export DBT_SCHEMA=analysis_crm_sales &&\n            cd /opt/airflow/dbt &&\n            /home/airflow/.local/bin/dbt --no-write-json test --target dev --models crm_stg_accounts\n            ']
[2024-07-15T23:59:22.508+0000] {subprocess.py:86} INFO - Output:
[2024-07-15T23:59:23.441+0000] {subprocess.py:93} INFO - [0m23:59:23  Running with dbt=1.8.3
[2024-07-15T23:59:23.554+0000] {subprocess.py:93} INFO - [0m23:59:23  Registered adapter: ***=1.8.2
[2024-07-15T23:59:23.722+0000] {subprocess.py:93} INFO - [0m23:59:23  Found 6 models, 4 sources, 652 macros
[2024-07-15T23:59:23.723+0000] {subprocess.py:93} INFO - [0m23:59:23
[2024-07-15T23:59:23.723+0000] {subprocess.py:93} INFO - [0m23:59:23  Nothing to do. Try checking your model configs and model specification args
[2024-07-15T23:59:24.138+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-15T23:59:24.148+0000] {__init__.py:75} DEBUG - Lineage called with inlets: [], outlets: []
[2024-07-15T23:59:24.149+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_stg_accounts manual__2024-07-15T23:54:38.113182+00:00 [running]> from DB
[2024-07-15T23:59:24.153+0000] {taskinstance.py:1356} DEBUG - Clearing next_method and next_kwargs.
[2024-07-15T23:59:24.153+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=run_dbt_model_crm_sales, task_id=test.matt_demo.crm_stg_accounts, execution_date=20240715T235438, start_date=20240715T235922, end_date=20240715T235924
[2024-07-15T23:59:24.153+0000] {taskinstance.py:2305} DEBUG - Task Duration set to 1.719939
[2024-07-15T23:59:24.158+0000] {cli_action_loggers.py:83} DEBUG - Calling callbacks: []
[2024-07-15T23:59:24.202+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-07-15T23:59:24.224+0000] {dagrun.py:697} DEBUG - number of tis tasks for <DagRun run_dbt_model_crm_sales @ 2024-07-15 23:54:38.113182+00:00: manual__2024-07-15T23:54:38.113182+00:00, state:running, queued_at: 2024-07-15 23:54:38.125855+00:00. externally triggered: True>: 11 task(s)
[2024-07-15T23:59:24.224+0000] {dagrun.py:718} DEBUG - number of scheduleable tasks for <DagRun run_dbt_model_crm_sales @ 2024-07-15 23:54:38.113182+00:00: manual__2024-07-15T23:54:38.113182+00:00, state:running, queued_at: 2024-07-15 23:54:38.125855+00:00. externally triggered: True>: 2 task(s)
[2024-07-15T23:59:24.224+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-07-15T23:59:24.225+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [None]> dependency 'Trigger Rule' PASSED: False, Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 5 non-success(es). upstream_states=_UpstreamTIStates(success=5, skipped=0, failed=0, upstream_failed=0, removed=0, done=5), upstream_task_ids={'test.matt_demo.crm_performance', 'test.matt_demo.crm_stg_sales_teams', 'model.matt_demo.crm_stg_accounts', 'test.matt_demo.crm_stg_sales_pipelines', 'model.matt_demo.crm_stg_sales_pipelines', 'model.matt_demo.crm_performance', 'model.matt_demo.crm_stg_products', 'test.matt_demo.crm_stg_products', 'model.matt_demo.crm_stg_sales_teams', 'test.matt_demo.crm_stg_accounts'}
[2024-07-15T23:59:24.225+0000] {taskinstance.py:1093} DEBUG - Dependencies not met for <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [None]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 5 non-success(es). upstream_states=_UpstreamTIStates(success=5, skipped=0, failed=0, upstream_failed=0, removed=0, done=5), upstream_task_ids={'test.matt_demo.crm_performance', 'test.matt_demo.crm_stg_sales_teams', 'model.matt_demo.crm_stg_accounts', 'test.matt_demo.crm_stg_sales_pipelines', 'model.matt_demo.crm_stg_sales_pipelines', 'model.matt_demo.crm_performance', 'model.matt_demo.crm_stg_products', 'test.matt_demo.crm_stg_products', 'model.matt_demo.crm_stg_sales_teams', 'test.matt_demo.crm_stg_accounts'}
[2024-07-15T23:59:24.225+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-07-15T23:59:24.226+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_performance manual__2024-07-15T23:54:38.113182+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-07-15T23:59:24.226+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_performance manual__2024-07-15T23:54:38.113182+00:00 [None]> dependency 'Trigger Rule' PASSED: False, Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=0, skipped=0, failed=0, upstream_failed=0, removed=0, done=0), upstream_task_ids={'model.matt_demo.crm_performance'}
[2024-07-15T23:59:24.226+0000] {taskinstance.py:1093} DEBUG - Dependencies not met for <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_performance manual__2024-07-15T23:54:38.113182+00:00 [None]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=0, skipped=0, failed=0, upstream_failed=0, removed=0, done=0), upstream_task_ids={'model.matt_demo.crm_performance'}
[2024-07-15T23:59:24.227+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.test.matt_demo.crm_performance manual__2024-07-15T23:54:38.113182+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-07-15T23:59:24.228+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
